test/discovery/HazelcastCloudDiscoveryTest.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var HazelcastClient = require("../../").Client;
+var expect = require('chai').expect;
+
+var ClientConfig = require("../../lib/config/Config").ClientConfig;
+
+describe('HazelcastCloudDiscovery Test', function () {
+
+    // add token and group name and activate the test
+    xit('basic cloud discovery test', function () {
```

Comment:
Doesn't `xit` skip the test?

Reply:
I was told that it's been decided not to place basic functionality test in the PR builder for Java client too. QE team is going to test it. Given that I believe there must not be a skipped test in the codebase. I'd prefer removing this test since it is not going to be used.